### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by wkchanG7309**

### CODE REVIEW

File: mcr.cpp [hunk 1]
```diff
@@ -3,13 +3,13 @@ bool isWin(char game[3][3]){
 
 bool win = false; 
 
-if (game[0][0] == game[0][1] && game[0][1] == game[0][2] && (game[0][0] == 'X' || game[0][0] == 'O')) win = true;
```
Comment (wkchanG7309) [inline mcr.cpp]: Press the "+" button and add your comment to this particular line 6.

Comment (Wsine) [inline mcr.cpp]: @wkchanG7309  Press "+" button and add you comment to this particular line 6.

File: mcr.cpp [hunk 2]
```diff
@@ -3,13 +3,13 @@ bool isWin(char game[3][3]){
 
 bool win = false; 
 
-if (game[0][0] == game[0][1] && game[0][1] == game[0][2] && (game[0][0] == 'X' || game[0][0] == 'O')) win = true; 
+if (game[0][0] === game[0][1] && game[0][1] == game[0][2] && (game[0][0] == 'X' || game[0][0] == 'O')) win = true; 
 
-if (game[1][0] == game[1][1] && game[1][1] == game[1][2] && (game[1][0] == 'X' || game[1][0] == 'O')) win = true; 
+if (game[1][0] === game[1][1] && game[1][1] == game[1][2] && (game[1][0] == 'X' || game[1][0] == 'O')) win = true; 
 
-if (game[2][0] == game[2][1] && game[2][1] == game[2][2] && (game[2][0] == 'X' || game[2][0] == 'O')) win = true;
```
Comment (wkchanG7309) [inline mcr.cpp:10]: === is not supported by C++

Comment (wkchanG7309) [reply mcr.cpp:10]: Confirmed.

**Pull request merged**